src/components/SavedMarkdownList.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 
-export default function MarkdownTitles({ titles }) {
+export function MarkdownTitles({ titles }) {
 
   const listOfTitles = titles.map((title, i)  => {
     return <li key={i}> {title}</li>;
```

Comment:
```suggestion
    return <li key={i}>{title}</li>;
```
Keep that whitespace consistent!